app/controllers/v0/message_drafts_controller.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+module V0
+  class MessageDraftsController < SMController
+    def create
+      params = draft_create_params
+      draft = MessageDraft.new(params)
+      raise Common::Exceptions::ValidationErrors, draft unless draft.valid?
+      response = client.post_create_message_draft(params)
+
+      render json: response,
+             serializer: MessageSerializer,
+             meta:  {}
+    end
+
+    def update
+      params = draft_update_params
+      draft = MessageDraft.new(params)
+      raise Common::Exceptions::ValidationErrors, draft unless draft.valid?
+
+      response = client.post_create_message_draft(params)
+
+      render json: response,
+             serializer: MessageSerializer,
+             meta:  {}
+    end
+
+    private
+
+    def draft_create_params
+      # Call to MHV message create fails if unknown field present, and does not accept recipient_id. This
+      # functionality will be moved into 'gem' once gem is moved to vets-api.
+      params.require(:message_draft).permit(:category, :body, :recipient_id, :subject)
+    end
+
+    def draft_update_params
+      # Call to MHV message create fails if unknown field present, and does not accept recipient_id. This
+      # functionality will be moved into 'gem' once gem is moved to vets-api.
+      params.require(:message_draft).permit(:id, :category, :body, :recipient_id, :subject)
```

Comment:
Why are these comments necessary? I think it's very clear what params.require is doing and how it's being used without the need for comments, no?